### PR TITLE
v2.6 Installation notes

### PR DESCRIPTION
**Pull request by manuwelakanade-GSLAB**
v2.6 Installation notes

### CODE REVIEW

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan :  Can you please review and merge this request. Thanks.

File: docs/install.rst [hunk 1]
```diff
@@ -80,6 +80,8 @@ Uninstall F5 ACI ServiceCenter
 
   - It is important to disable the F5 ACI ServiceCenter first and then delete it, in order to clean out all the filesystem folders related to the application. To check whether the deletion of the app has happened correctly, please check /data2/gluster/gv0/ folder and ensure that it does not contain F5Networks_F5ACIServiceCenter folder on any of the APIC nodes. Not disabling the app first may lead to 'InvalidPaddingError' errors when performing app operations.
 
+  - If the error 'InvalidPaddingError' is observed, uninstall and re-install the application by following the aforementioned steps. If the database is to be retained, then export the database, uninstall and re-install followed by an import of the database. For more details check **Database Export and Import** section.
```
Comment (JosephPJordan) [inline docs/install.rst:83]: If the database is to be retained, export the database, uninstall and re-install, and then import the database. For more details check  the **Database Export and Import** section.

File: docs/faq.rst [hunk 2]
```diff
@@ -54,6 +54,11 @@ The F5 ACI ServiceCenter UI will show the last 100 errors or warnings which were
 
 In order to recreate the deleted log files, disable and re-enable the application from the APIC **Apps** tab. Once the app is re-enabled, the logging should work correctly. 
 
+------
+
+**Q. Why do I see 'Request timeout' error on the F5 ACI ServiceCenter UI?**
```
Comment (JosephPJordan) [inline docs/faq.rst:59]: ...I see a 'Request timeout'.. error

File: docs/faq.rst [hunk 3]
```diff
@@ -54,6 +54,11 @@ The F5 ACI ServiceCenter UI will show the last 100 errors or warnings which were
...
+
+Application UI may show the 'Request timeout' error, if the application or APIC is receiving a lot of traffic. Users can retry the same operation again (which displayed the error) and it should be successful after one or more retries.
```
Comment (JosephPJordan) [inline docs/faq.rst:62]: The Application UI may...
You can retry the same operation that displayed the error and it should be successful after one or more retries.

**Comment by JosephPJordan** [comment]
@manuwelakanade-GSLAB, a couple minor changes requested

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Thanks for the review. I have addressed the comments and updated the PR.

**Review by JosephPJordan — APPROVED** [review]

**Pull request merged**